politeiawww/api/v1/v1.go

```diff
@@ -710,3 +714,42 @@ type UserCommentsVotes struct{}
 type UserCommentsVotesReply struct {
 	CommentsVotes []CommentVote `json:"commentsvotes"`
 }
+
+// VoteOptionResult describes a VotingOption info
+// along with the total votes it received
+type VoteOptionResult struct {
+	Option        VoteOption `json:"option"`        // Vote Option
+	VotesReceived uint64     `json:"votesreceived"` // Votes received by the option
+}
+
+// CastedVotesSummary describes the total votes for a proposal
+// and the VoteOptionResult for each available voting option
+type CastedVotesSummary struct {
```

Comment:
It looks like we may not need this struct and include the contents directly to ProposalVotingStatusReply. Don't create new structs unless there is a reason to (like a tuple or an array is a reason).